e2e/cypress/integration/channel/message_edit_post_with_attachment_spec.js

```diff
@@ -11,8 +11,9 @@ import * as TIMEOUTS from '../../fixtures/timeouts';
 
 describe('MM-13697 Edit Post with attachment', () => {
     before(() => {
-        // # Login and go to /
+        // # Login, set display preference and go to /
         cy.apiLogin('user-1');
+        cy.apiSaveMessageDisplayPreference();
```

Comment:
This test is failing when other test failed and didn't recover from resetting display preference.